### ISSUE TITLE
Feature Request: Optionally Set qrCode output format/quality (filesize/bandwidth improvement)

### ISSUE THREAD

**Issue by sandro-salles** [open-wa/wa-automate-nodejs#639]
Allowing to set qrCodeOutputFormat and qrCodeOutputQuality would help with the size of generated qrcodes, specially for those who don't need a big full quality png and have to transfer the file/dataurl over the network to another services.

canvas.toDataURL defaults to png, so the ideia is to be able to set a different output format (jpeg or webm) and the output quality (eg. 0.5).

Benchmarks:
https://www.import.io/post/html5-canvas-todataurl-webm-vs-png-vs-jpeg/

Code example for **auth.ts/retrieveQR**:

```
const qrCode = await waPage.evaluate(
   `document.querySelector("canvas[aria-label='Scan me!']").toDataURL('image/${config.qrCodeOutputFormat}', ${config.qrCodeOutputQuality})`
);
```

Regards!

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.98](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.97...1.9.98) (2020-07-12)

- build(deps-dev): bump release-it from 13.6.4 to 13.6.5 [`#637`](https://github.com/open-wa/wa-automate-nodejs/pull/637)
- Release 1.9.98 [`a4c8118`](https://github.com/open-wa/wa-automate-nodejs/commit/a4c8118db985a75668d90b64bc37eb6b981d665f)
- Refactored config and implemented qr config options #639 [`c2ed7b7`](https://github.com/open-wa/wa-automate-nodejs/commit/c2ed7b7f654ab47b97d2a0e76cab0dedcd2b97d2)
- Prelim work for aliases #583 [`4c3bfc7`](https://github.com/open-wa/wa-automate-nodejs/commit/4c3bfc702244420e5e4649af0dd208c0bcb18868)
- Fix spinnies errors [`724c9a1`](https://github.com/open-wa/wa-automate-nodejs/commit/724c9a1eb376b22aa98d20208243c9948bafd53d)
- Update pop up title [`9d54d80`](https://github.com/open-wa/wa-automate-nodejs/commit/9d54d805ca60d11d675894a75ea8048423dddc1b)

**Comment by smashah** [comment]
This has been implemented now

```javascript
//https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#qrquality
qrQuality: QRQuality.FIVE,
//https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#qrformat
qrFormat: QRFormat.JPEG,
```
